src/components/app/details/appDetails/DeploymentStatusBreakdown.tsx

```diff
@@ -51,19 +57,23 @@ export default function DeploymentStatusDetailBreakdown({
         )
     }
     return (
-        <div className="deployment-status-breakdown-container pl-20 pr-20">
-            {deploymentStatusDetailsBreakdownData.deploymentError && (
-                <InfoColourBar
-                    message={deploymentStatusDetailsBreakdownData.deploymentError}
-                    classname="error_bar cn-9 mb-20 lh-20"
-                    Icon={Error}
-                    iconClass="icon-dim-20"
-                />
-            )}
-            {renderStatusDetailRow('DEPLOYMENT_INITIATED')}
-            {renderStatusDetailRow('GIT_COMMIT')}
-            {renderStatusDetailRow('KUBECTL_APPLY')}
-            {renderStatusDetailRow('APP_HEALTH', true)}
-        </div>
+        <>
+           {url.includes('/cd-details') ? null :  <ErrorBar appDetails={_appDetails} /> }
```

Comment:
use the existing const for `/cd-details` also try to use location.pathname instead of whole url

Reply:
And make use of `&&`